### DISCUSSION TITLE
Getting a Warning in my console about `form-control-focus()` being removed in Bootstrap 5

### DISCUSSION THREAD

**Discussion by torressam333** [twbs/bootstrap#33366] [Q&A] [answered]
Hello, I am getting this warning when running `npm run dev` 

```
WARNING The `form-control-focus()` mixin has been deprecated as of v4.4.0. It will be removed entirely in v5.
```

My question is: will there be a replacement for this in 4.5 and up? 

Thank you.

**Comment by ffoodd** [comment, marked as answer]
It's dropped in v5 only, as mentionned in the warning. You're safe with any upcoming v4 🙂
